apps/src/gamelab/GameLab.js

```diff
@@ -1194,6 +1200,33 @@ GameLab.prototype.onP5Draw = function () {
     this.drawInProgress = true;
     if (getStore().getState().runState.isRunning) {
       this.eventHandlers.draw.apply(null);
+      if (this.level.validationCode) {
+        try {
+          const validationResult =
+            this.JSInterpreter.interpreter.marshalInterpreterToNative(
+              this.JSInterpreter.evalInCurrentScope(`
+                __validationState = null;
+                __validationResult = null;
+                ${this.level.validationCode}
+                ret = {
```

Comment:
Where does the magic `ret` come from?

Reply:
Ah, I was trying to workaround a bug our intepreter (or rather the acorn parser?) that appears to choke on bare objects used as standalone expressions. Updated to wrap the code in a function and properly return the value I care about.